go-controller/pkg/ovndbmanager/ovndbmanager.go

```diff
@@ -209,3 +210,16 @@ func ensureClusterRaftMembership(db string, kclient kube.Interface) {
 		}
 	}
 }
+
+// EnableDBMemTrimming enables memory trimming on DB compaction
+func EnableDBMemTrimming() error {
```

Comment:
Can you include as a comment what the downstream impact of this is? 

What does the trimming on db compaction do? Is the compaction happening every 10 mins and how often does the memory re-claiming happen? there is a potentially periodic go-routine that we need to run as well?

Reply:
I'll include a comment. I dont think we need to run in a go-routine. Nothing should turn this off after we configure it.